app/Resources/translations/default/AdminGlobal.xlf

```diff
@@ -2063,16 +2059,16 @@ File: modules/ps_buybuttonlite/ps_buybuttonlite.php:168</note>
       <trans-unit id="318b2739ddc2c16c97b33c9b04b79f3e">
         <source>Me</source>
         <target>Me</target>
-        <note>Line: 50</note>
+        <note>Line: 51</note>
       </trans-unit>
     </body>
   </file>
-  <file original="src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/payments.html.twig" source-language="en-US" target-language="en" datatype="plaintext">
+  <file original="src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/shipping.html.twig" source-language="en-US" target-language="en" datatype="plaintext">
```

Comment:
No relation between those two, `Card number` wasn't supposed to be in this domain (it's in `AdminOrderscustomersFeature`), whereas `Gift message:` has just been added